packages/checkbox/src/CheckboxMixin.ts

```diff
@@ -71,6 +78,9 @@ export function CheckboxMixin<T extends Constructor<ReactiveElement>>(
         protected render(): TemplateResult {
             return html`
                 <input
+                    aria-label="${ifDefined(
+                        this.label || this.ariaLabel || undefined
```

Comment:
Yes, the component supports ariaLabel implicitly. I can't take a screenshot of the popover with that information from my ide, but it redirects to a document explaining the internals https://wicg.github.io/aom/explainer.html#default-semantics-for-custom-elements-via-the-elementinternals-object

Reply:
Will `this.ariaLabel` ever be defined? I don't see it as a property on `CheckboxMixin`

Reply:
if you do <sp-checkbox aria-label="Test"></sp-checkbox> then it will be used. Perhaps it's better to rely only on `aria-label` instead of `label`. That won't require any changes on the checkbox at all. What do you think?

Reply:
Yeah, that sounds good! Might be cleaner that way.